filterExprParser_test.go

```diff
@@ -206,7 +206,7 @@ func TestFilterExpressionParser(t *testing.T) {
 	userData = map[string]interface{}{
 		"onePath": map[string]interface{}{
 			"field1": -2,
-			"field2": 2,
+			"field2": 2e30,
 		},
```

Comment:
i don't think this will overflow int64. 2e60 could.

Reply:
Largest int64 is 9223372036854775808
2e20 could overflow it.

Reply:
2e30 is 2*e^30, correct? int64 can hold 2^63 even without the sign bit.  e<4, so e30 < 2^61

Reply:
The e is scientific notation and not the euler's number (I think that's what you mean?)
https://stackoverflow.com/questions/13617818/json-e-and-json-e